stable/grafana/templates/NOTES.txt

```diff
@@ -20,9 +20,9 @@
    NOTE: It may take a few minutes for the LoadBalancer IP to be available.
         You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "grafana.fullname" . }}'
      export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "grafana.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
-     http://$SERVICE_IP:{{ .Values.service.port -}}
+     {{ (index .Values "grafana.ini").server.protocol }}://$SERVICE_IP:{{ .Values.service.port -}}
 {{ else if contains "ClusterIP"  .Values.service.type }}
-     export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "grafana.fullname" . }},component={{ .Values.name }}" -o jsonpath="{.items[0].metadata.name}")
+     export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "grafana.name" . }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
Another dumb question perhaps, but why was this changed?

Reply:
This is a bug in the official chart.  The pods aren't labelled with component, and the app they are labelled with is from grafana.name not grafana.fullname